primitives/trie/src/storage_proof.rs

```diff
@@ -176,7 +172,7 @@ impl CompactProof {
 /// An iterator over trie nodes constructed from a storage proof. The nodes are not guaranteed to
 /// be traversed in any particular order.
```

Comment:
Comment could be updated : "in lexicographical order of encoded nodes.", or something like that.

Reply:
Let's keep it that way, because we then can change this later.